documentation/source/user_section/tutorials/spinalcord-segmentation/fixing-failed-segmentations.rst

```diff
@@ -1,11 +1,15 @@
-.. _correcting_sct_propseg:
+Fixing a failed segmentation
```

Comment:
i'm afraid many people will go to this page and try to fix their failed segmentation from deepseg_sc (and in the future, deepseg). I would make it explicit in the title/URL that this is specific to propseg. Eg: fixing-failed-propseg-segmentation